variables.tf

```diff
@@ -132,6 +132,23 @@ variable "configuration" {
   default = null
 }
 
+variable "admin_pass" {
+  type        = string
+  description = "The password for the database administrator. If not specified, an empty string is provided for the password and the user ID cannot be used. In this case, more users must be specified in a user block. The user password must be in the range of 10-32 characters."
```

Comment:
"In this case, more users must be specified in a user block" <-- To me this sound like if no value is passed for `admin_pass` then a value must be passed for the `users` variable? But is that really true? Can they both not be null / empty list ?

Reply:
I am not sure about this, I picked this text from the IBM terraform provider documentation. Let me have a look into this and get back to you

Reply:
I did more research on this. 

Each admin_pass comes along with one admin User ID. If there is no value passed for the admin_pass then it will take an empty string for the password and the "admin" USER ID cannot be used. You must set an admin_pass to use the admin USER ID. (https://cloud.ibm.com/docs/databases-for-postgresql?topic=databases-for-postgresql-user-management&interface=ui#user-admin)

It means that if the user wants to add more users then a "user" block is required to do so.

So the `admin_pass` and `users` can both be null at the same time.

I will change this sentence to

```
The password for the database administrator. If the admin password is null then the admin user ID cannot be accessed. More users can be specified in a user block. The admin password must be in the range of 10-32 characters.
```